src/util/tracing.ml

```diff
@@ -86,7 +86,7 @@ let traceTag (sys : string) : Pretty.doc =
   (text ((ind !indent_level) ^ "%%% " ^ sys ^ ": "))
 
 let printtrace sys d: unit =
-  fprint stderr ~width:80 ((traceTag sys) ++ d);
+  fprint stderr ~width:max_int ((traceTag sys) ++ d);
```

Comment:
Yes, such changes have been made to many other uses of `~width` arguments of `Pretty`. Otherwise `Pretty` tries (and quite poorly at that) to wrap lines at 80 characters, which is completely unnecessary when tracing output is forwarded to a file (which is usually the case). It makes diffing such tracing output files extremely annoying since most of the lines could be changed while the only difference is in extra newlines (which whitespace-ignoring diff doesn't ignore either).

It's much more reasonable to just configure your editor whether you want line wraps or not (based on the actual screen width), rather than assuming 80, which the long tracing output almost always exceeds as well, so it doesn't even achieve the desired wrapping.